<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>wybierz klase lotu:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="firstBox">
      <property name="text">
       <string>Pierwsza klasa 300zl</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bussinesBox">
      <property name="text">
       <string>biznes klasa 150zl</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="economicBox">
      <property name="text">
       <string>Klasa ekonomiczna 20zl</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
